src/components/TodoList.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/react-in-jsx-scope */
+import TodoNote from "./TodoNote";
+import "../index.css";
+import React, {useState, useEffect, useRef} from "react";
+import {nanoid} from "nanoid";
```

Comment:
Why is this not included in the package.json?

Reply:
Done.